### ISSUE TITLE
error: can't copy '../LICENSE': doesn't exist or not a regular file while running setup.py

### ISSUE THREAD

**Issue by devz3r0** [pretalx/pretalx#844]
## Current Behavior
```
 error: can't copy '../LICENSE': doesn't exist or not a regular file

    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-dk53wq1b/pretalx/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-z6q4t5ty/install-record.txt --single-version-externally-managed --compile --user --prefix=" failed with error code 1 in /tmp/pip-install-dk53wq1b/pretalx/
```

## Expected Behavior
copying the LICENSE file

## Steps to Reproduce

1. do a fresh install with debian 10
2. follow the manual at https://docs.pretalx.org/en/latest/administrator/installation.html
3. get to the step: sudo -u pretalx pip3 install --user -U pretalx[redis]
4. crash

## Context

googling after this issue brought me to this:
https://github.com/ilius/pyglossary/issues/77

so is it possible that a similar issue is here?

## Your Environment

fresh Debian 10 VM in Virtualbox
Linux debian10-pretalx 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2+deb10u2 (2019-11-11) x86_64 GNU/Linux

* Version used:
python3 --version
Python 3.7.3

* Environment name and version (e.g. Chrome 39, python 3.5):
* Operating System and version (desktop or mobile):
* Link to your instance, if in production:

**Comment by rixx** [comment]
No, this is due to the change made in 39ea6a4, in response to #746. Apparently I misunderstood how to do this with setuptools.

**Comment by devz3r0** [comment]
is it possible, that in the redis version the file is missing?

**Comment by rixx** [comment]
The workaround is to install pretalx via git, I'll release a bugfix release in a couple of days.

```
pip3 install --user -U "git+git://github.com/pretalx/pretalx.git@v1.2.0#egg=pretalx&subdirectory=src"
```

**Comment by devz3r0** [comment]
Hi,

I tried the worksount out and got:
```
 sudo -u pretalx pip3 install --user -U "git+git://github.com/pretalx/pretalx.git@v1.2.0#egg=pretalx&subdirectory=src"
```

```
Collecting pretalx from git+git://github.com/pretalx/pretalx.git@v1.2.0#egg=pretalx&subdirectory=src
  Cloning git://github.com/pretalx/pretalx.git (to revision v1.2.0) to /tmp/pip-install-5jskn86f/pretalx
  Did not find branch or tag 'v1.2.0', assuming revision or ref.
error: Pfadspezifikation 'v1.2.0' stimmt mit keinen git-bekannten Dateien überein.
Command "git checkout -q v1.2.0" failed with error code 1 in /tmp/pip-install-5jskn86f/pretalx
```
is there something else to adjust?

**Comment by rixx** [comment]
Sorry, I meant v1.1.0 of course, since 1.2.0 is not out yet.

**Comment by rixx** [comment]
I just released v1.1.1 which should fix this issue.
